templated_mail/mail.py

```diff
@@ -6,6 +6,10 @@
 from django.views.generic.base import ContextMixin
 
 
+SETTINGS_DOMAIN = 'TEMPLATED_MAIL_DOMAIN'
```

Comment:
Isn't it a little confusing? I understand why we sometimes create pseudo-constants like this. But here we use one name in the settings file and another one in the code. At least I felt confused at first here.